### PR TITLE
[OSDEV-1482] GET api/v1/moderation-events/{moderation_id} should return single response instead of array

### PR DESCRIPTION
**Pull request by roman-stolar**
[OSDEV-1482](https://opensupplyhub.atlassian.net/browse/OSDEV-1482) GET api/v1/moderation-events/{moderation_id} should return single response instead of array

- Updated response format and added handling for 404

[OSDEV-1482]: https://opensupplyhub.atlassian.net/browse/OSDEV-1482?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by mazursasha1990 — COMMENTED** [review]
Are there test cases for these changes?

**Comment by roman-stolar** [comment]
> Are there test cases for these changes?

Yes, I will update integration test for it.

**Review by protsack-stephan — DISMISSED** [review]
Looks good to me!

File: doc/release/RELEASE-NOTES.md [hunk 1]
```diff
@@ -49,6 +49,7 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 
 ### Bugfix
 * [OSDEV-1388](https://opensupplyhub.atlassian.net/browse/OSDEV-1388) - The waiter from boto3 cannot wait more than half an hour so we replaced it with our own.
+* [OSDEV-1482](https://opensupplyhub.atlassian.net/browse/OSDEV-1482) - The `GET api/v1/moderation-events/{moderation_id}` endpoint return single response instead of array with one item.
```
Comment (VadimKovalenkoSNF) [inline doc/release/RELEASE-NOTES.md:52]: Fix grammar: endpoint `returns`

**Review by VadimKovalenkoSNF — CHANGES_REQUESTED** [review]
Minor fix required.

File: doc/release/RELEASE-NOTES.md (hunk 1 above)
Comment (roman-stolar) [reply doc/release/RELEASE-NOTES.md]: Resoved.

**Review by VadimKovalenkoSNF — APPROVED** [review]
Approved.

**Review by protsack-stephan — APPROVED** [review]

**Review by vladsha-dev — APPROVED** [review]
LGTM

**Review by VadimKovalenkoSNF — APPROVED** [review]
LGTM

**Pull request merged**